examples/src/main/java/com/apple/foundationdb/record/sample/Main.java

```diff
@@ -455,7 +455,7 @@ public static void main(String[] args) {
                     SampleProto.Customer.Builder builder = SampleProto.Customer.newBuilder().mergeFrom(record.getRecord());
                     addressMap.put(builder.getFirstName() + " " + builder.getLastName(),
                             builder.getEmailAddressList());
-                });
+                }).join();
```

Comment:
Again, I think we'd like to encourage our standard mechanisms for waiting for asynchronous work to complete, so I'd do something like:

```
cx.asyncToSync(FDBStoreTimer.Waits.WAIT_EXECUTE_QUERY, 
    cursor.forEach((FDBQueriedRecord<Message> record) -> {
            ...
         }));
```